tests/e2e/preferential_topology.go

```diff
@@ -182,10 +182,6 @@ var _ = ginkgo.Describe("[Preferential-Topology] Preferential-Topology-Provision
 		err = createTagForPreferredDatastore(masterIp, allowedTopologyRacks)
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 
-		framework.Logf("Waiting for %v for preferred datastore to get refreshed in the environment",
```

Comment:
why are we removing this?

Reply:
This is not required. We need to wait only when we are taking any new datastore preference. Here we are doing cleanup and recreating preferred datastore tags only for the next testcase to pick it up.